localstack/logging/setup.py

```diff
@@ -78,6 +79,23 @@ def setup_logging_from_config():
         for name, level in trace_internal_log_levels.items():
             logging.getLogger(name).setLevel(level)
 
+    if raw_value := config.LOGGING_OVERRIDE:
+        try:
+            logging_overrides = json.loads(raw_value)
```

Comment:
WDYT json vs key-value pairs? I think the latter might be easier to write, and we do not need nested structures. I do not have a strong opinion though.

Reply:
I think that's a good usability improvement :+1:

Reply:
Implemented in db163efc557d2a75ca5798875142e0534a1aa4c1